third-party/realdds/src/dds-stream.cpp

```diff
@@ -4,10 +4,16 @@
 #include <realdds/dds-stream.h>
 #include "dds-stream-impl.h"
 
+#include <realdds/dds-topic.h>
+#include <realdds/dds-topic-reader.h>
+#include <realdds/dds-subscriber.h>
+#include <realdds/topics/image/image-msg.h>
 #include <realdds/dds-exceptions.h>
 
+#include <fastdds/dds/subscriber/SampleInfo.hpp>
```

Comment:
Not used?

Reply:
Used in `handle_frames`

Reply:
But not used there, either... it's optional. But OK let's keep it maybe later it'll be used.